test/ui-test/src/screens/StatusWalletScreen.py

```diff
@@ -199,6 +206,30 @@ def delete_saved_address(self, name: str):
         
         click_obj_by_name(SavedAddressesScreen.DELETE.value)
         click_obj_by_name(SavedAddressesScreen.CONFIRM_DELETE.value)
+    
+    def toggle_network(self, network_name: str):
+        click_obj_by_name(MainWalletScreen.NETWORK_SELECTOR_BUTTON.value)
+
+        list = get_obj(NetworkSelectorPopup.LAYER_1_REPEATER.value)
+        for index in range(list.count):
+            if list.itemAt(index).objectName == network_name:
+                click_obj(list.itemAt(index))
+                click_obj_by_name(MainWalletScreen.ACCOUNT_NAME.value)
+                time.sleep(2)
```

Comment:
Why is this needed?

Reply:
Because switching network takes time, there is async task doing all the recalculation

Reply:
And could we `squish.waitFor` something instead??